README.md

```diff
@@ -113,6 +113,7 @@ _Web Analytics_
 - [Countly](https://count.ly) - Real time mobile and web analytics, crash reporting and push notifications platform. ([Source Code](https://github.com/countly)) `AGPL-3.0` `Javascript`
 - [Druid](http://druid.io/) - Distributed, column-oriented, real-time analytics data store. ([Source Code](https://github.com/druid-io/druid)) `Apache-2.0` `Java`
 - [Fathom Analytics](https://usefathom.com) - Simple & trustworthy website analytics. ([Source Code](https://github.com/usefathom/fathom)) `MIT` `Go`
+- [Freshlytics](https://github.com/sheshbabu/freshlytics/blob/master/docs/screenshots.md) - Privacy respecting, cookie free and low resource usage analytics platform. ([Source Code](https://github.com/sheshbabu/freshlytics)) `MIT` `TypeScript`
```

Comment:
Please change the main link to https://github.com/sheshbabu/freshlytics and remove the Source code link